composer.json

```
@@ -20,6 +20,7 @@
         }
     },
     "require": {
+        "php": "^7.2",
```

Zouden we hier geen 7.3 gebruiken?
```suggestion
        "php": "^7.3",
```

---

Updated.